domains/grid/components/GridFloatingMenu.vue

```diff
@@ -137,7 +138,7 @@ onUnmounted(() => {
 
 <template>
   <div
-    v-if="isConnected"
+    v-if="isConnected && isConnectedUserViewingOwnProfile"
```

Comment:
LGTM, I only suggest renaming `isConnectedUserViewingOwnProfile` property to a shorter version if there's no other conflicting context that could be misleading. I think `isViewingOwnProfile` carries the same meaning and is shorter.
```suggestion
    v-if="isConnected && isViewingOwnProfile"
```

Reply:
I like descriptive names even that they seem to long so just by looking on them you get exact context. I'm fine for `isViewingOwnProfile` although it's not straight what type of "viewer" we deal here.